apps/src/blockly/types.ts

```diff
@@ -144,14 +143,14 @@ export interface BlocklyWrapperType extends GoogleBlocklyType {
   ) => void;
   setInfiniteLoopTrap: () => void;
   clearInfiniteLoopTrap: () => void;
-  getInfiniteLoopTrap: () => string;
+  getInfiniteLoopTrap: () => string | null;
   loopHighlight: (apiName: string, blockId: string) => string;
   getWorkspaceCode: () => string;
   addChangeListener: (
     blockspace: GoogleBlockly.Workspace,
     handler: (e: GoogleBlockly.Events.Abstract) => void
   ) => void;
-  getGenerator: () => JavascriptGeneratorType;
+  getGenerator: () => ExtendedJavascriptGenerator | javascriptGeneratorType;
```

Comment:
can it return either type or will this always return `ExtendedJavascriptGenerator`?